app/api/models/errors/MtdErrors.scala

```diff
@@ -211,28 +211,24 @@ object V2_RangeToDateBeforeFromDateError
     )
 // Invalid Errors
 
-object InvalidDocNumberError extends MtdError(code = "INVALID_DOC_NUMBER", message = "The provided docNumber is invalid.")
+object DocNumberFormatError extends MtdError(code = "FORMAT_DOC_NUMBER", message = "The provided docNumber is invalid")
 
-object InvalidOnlyOpenItemsError extends MtdError(code = "INVALID_ONLY_OPEN_ITEMS", message = "The provided onlyOpenItems is invalid.")
+object OnlyOpenItemsFormatError extends MtdError(code = "FORMAT_ONLY_OPEN_ITEMS", message = "The provided onlyOpenItems is invalid")
 
-object InvalidIncludeLocksError extends MtdError(code = "INVALID_INCLUDE_LOCKS", message = "The provided includeLocks is invalid.")
+object IncludeLocksFormatError extends MtdError(code = "FORMAT_INCLUDE_LOCKS", message = "The provided includeLocks is invalid")
 
-object InvalidCalculateAccruedInterestError
-    extends MtdError(code = "INVALID_CALCULATE_ACCRUED_INTEREST", message = "The provided calculateAccruedInterest is invalid.")
+object CalculateAccruedInterestFormatError
+    extends MtdError(code = "FORMAT_CALCULATE_ACCRUED_INTEREST", message = "The provided calculateAccruedInterest is invalid")
 
-object InvalidCustomerPaymentInformationError
-    extends MtdError(code = "INVALID_CUSTOMER_PAYMENT_INFORMATION", message = "The provided customerPaymentInformation is invalid.")
+object CustomerPaymentInformationFormatError
+    extends MtdError(code = "FORMAT_CUSTOMER_PAYMENT_INFORMATION", message = "The provided customerPaymentInformation is invalid")
 
-object InvalidDateFromError extends MtdError(code = "INVALID_DATE_FROM", message = "The provided dateFrom is invalid.")
+object RemovePaymentOnAccountFormatError extends MtdError(code = "FORMAT_REMOVE_PAYMENT_ON_ACCOUNT", message = "The provided removePOA is invalid")
 
-object InvalidDateToError extends MtdError(code = "INVALID_DATE_TO", message = "The provided dateTo is invalid.")
+object IncludeEstimatedChargesFormatError
+    extends MtdError(code = "FORMAT_INCLUDE_ESTIMATED_CHARGES", message = "The provided includeEstimatedCharges is invalid")
 
-object InvalidRemovePaymentOnAccountError extends MtdError(code = "INVALID_REMOVE_PAYMENT_ON_ACCOUNT", message = "The provided removePOA is invalid.")
-
-object InvalidIncludeChargeEstimateError
-    extends MtdError(code = "INVALID_INCLUDE_CHARGE_ESTIMATE", message = "The provided includeChargeEstimate is invalid.")
-
-object InvalidDateRangeError extends MtdError(code = "RULE_INVALID_DATE_RANGE", message = "The provided date range is invalid.")
+object InvalidDateRangeError extends MtdError(code = "RULE_INVALID_DATE_RANGE", message = "The provided date range is invalid")
```

Comment:
Have we considered deleting line 231? It is almost identical to line 164. The only difference is the error on line 164 is called **Rule**InvalidDateRangeError. It's also very similar to line 158...